src/java/org/apache/cassandra/tools/NodeProbe.java

```diff
@@ -313,10 +318,18 @@ protected void connect() throws IOException
                     "Invalid ObjectName? Please report this as a bug.", e);
         }
 
-        memProxy = ManagementFactory.newPlatformMXBeanProxy(mbeanServerConn,
-                ManagementFactory.MEMORY_MXBEAN_NAME, MemoryMXBean.class);
-        runtimeProxy = ManagementFactory.newPlatformMXBeanProxy(
-                mbeanServerConn, ManagementFactory.RUNTIME_MXBEAN_NAME, RuntimeMXBean.class);
+        try
```

Comment:
I just put this to try/catch so when it fails in in-jvm then we at least not fail completely